spec/factories/sender_infos.rb

```diff
@@ -0,0 +1,23 @@
+FactoryGirl.define do
+  factory :sender_info do
```

Comment:
Jak teraz na to patrzę, to `SenderInfo` i `RecipientInfo` są praktycznie identyczne. Być może dobrze byłoby opiąć je jednym modelem z enumeratorem decydującym czy jest to Sender, czy Recipient.